google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/wire/SubscriberImpl.java

```diff
@@ -115,13 +129,20 @@ protected void stop() {
 
   @Override
   public ApiFuture<Offset> seek(SeekRequest request) {
```

Comment:
Should we reset the tokenCounter on seeks()?

Reply:
Yes, almost certainly.

Reply:
Would we want to do this when a seek request is sent, or when a seek response is received?  If a client initiates a seek, and then immediately calls allowFlow before receiving a seek response, what is the expectation here?

Reply:
We should do it as soon as the client initiates the seek, IMO. If you think about the stream, messages are sent in order, so the flow-control should be processed after the seek.

Reply:
sounds good thanks!  I have added a tokenCounter reset on seek